_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -33,25 +35,22 @@ Components, and more specifically their HTML and Javascript files, live in packa
             "label": "Pinger",
             "path": "ping.html"
         }
-    },
-    "content-security-policy": "default-src 'self' 'unsafe-inline' 'unsafe-eval'"
+    }
 }
 ```
 
-The manifest above has a `"tools"` subsection. Each tool is listed in the *Tools* menu by Cockpit. The `"path"` is the name of the HTML file that implements the tool, and the `"label"` is the text to show in the *Tools* menu.
+The manifest above has a `"tools"` subsection, which means that it will appear in the lower section of the menu (see the [manifest documentation](https://cockpit-project.org/guide/latest/packages.html#package-manifest) for details). Each tool is listed in the menu by Cockpit. The `"path"` is the name of the HTML file that implements the tool, and the `"label"` is the text to show in the menu.
 
-You'll notice that we haven't told Cockpit about the package yet. To do so you either place or symlink the package into one of two places:
+You'll notice that we haven't told Cockpit about how to find the package yet. To do so you either place or symlink the package into one of two places:
 
- * `~/.local/share/cockpit`<br>
-In your home directory, for user specific packages, and ones that you're working on. You can edit these on the fly and just refresh your browser to see changes.
- * `/usr/share/cockpit`<br>
-For installed packages available to all users. These should not be changed while Cockpit is running.
+ * `~/.local/share/cockpit` in your home directory, for user specific packages, and ones that you're working on. You can edit these on the fly and just refresh your browser to see changes.
```

Comment:
```suggestion
 * `~/.local/share/cockpit` is in your home directory. It's used for user specific packages and ones that you're developing. You can edit these on the fly and refresh your browser to see changes.
```

Reply:
I'd drop the first "is", it's confusing IMHO.